awx/ui/client/src/inventories-hosts/inventories/inventory.list.js

```diff
@@ -93,6 +93,13 @@ export default ['i18n', function(i18n) {
 
         fieldActions: {
             columnClass: 'col-md-2 col-sm-3 col-xs-4',
+            network: {
+                label: i18n._('Network Visualization'),
+                ngClick: 'goToGraph(inventory)',
+                awToolTip: i18n._('Network Visualization'),
+                dataPlacement: 'top',
+                ngShow: '!inventory.pending_deletion && inventory.summary_fields.user_capabilities.edit'
```

Comment:
OK so just to confirm - there's no _read only_ view of the network topography?  A user must have edit permissions on the inventory to access this?

Reply:
Currently yes, but in the next few days I hope to change that so that the canvas has a readonly capability. More coming in the near future.